site/components/pages/Retirements/SingleRetirement/ProjectDetails/List/index.tsx

```diff
@@ -15,48 +15,63 @@ type Props = {
   projectLink: string;
   headline: string;
   totalRetired?: string;
+  currentSupply?: string;
   isMossOffset?: boolean;
+  methodology?: string;
+  location?: string;
 };
 
 export const ProjectDetail: FC<Props> = (props) => {
-  const { projectLink, headline, tokenAddress, totalRetired } = props;
   const { locale } = useRouter();
-
   const trimTotalRetired =
-    !!totalRetired && trimWithLocale(totalRetired, 2, locale);
+    props.totalRetired && Number(props.totalRetired)
+      ? trimWithLocale(props.totalRetired, 0, locale)
+      : null;
+  const trimCurrentSupply =
+    props.currentSupply && Number(props.currentSupply)
+      ? trimWithLocale(props.currentSupply, 0, locale)
+      : null;
 
   return (
     <div className={styles.listItem}>
       <Text t="h4">
-        <A className="link" href={projectLink}>
-          {headline}{" "}
+        <A className="link" href={props.projectLink}>
+          {props.headline}{" "}
           <span className="svg">
             <LaunchIcon fontSize="inherit" />
           </span>
         </A>
       </Text>
+      {props.location && (
+        <Text t="caption">
+          <Trans>Location: {props.location}</Trans>
+        </Text>
+      )}
+      {props.methodology && (
+        <Text t="caption">
+          <Trans>Methodology: {props.methodology}</Trans>
+        </Text>
+      )}
       {trimTotalRetired && (
-        <Text>
-          {trimTotalRetired}{" "}
-          <Trans id="retirement.single.project_details.tonnes">Tonnes</Trans>
+        <Text t="caption">
+          <Trans>{trimTotalRetired} Tonnes retired in total</Trans>
+        </Text>
+      )}
+      {trimCurrentSupply && (
+        <Text t="caption">
+          <Trans>{trimCurrentSupply} Tonnes remaining</Trans>
```

Comment:
All Single retirements do not show the values from the retirement  🦐
I think you need to extract the strings again.

And remove all dynamic values out of the `<Trans>` for an easier translator life.

For example [here](https://klimadao-site-git-atmos-page-performance-klimadao.vercel.app/retirements/atmosfearful.klima/1)

<img width="409" alt="Bildschirm­foto 2023-02-20 um 13 10 06" src="https://user-images.githubusercontent.com/95881624/220105052-a7965dcf-0e7f-4cc2-ba11-419cb7bba8a8.png">